styleselection.cpp error:ordered comparison palette->getStylePage(styleId)

Tried building from opentoonz/master today on OSX

Got the following error:

`
/Users/dave/Documents/Git/opentoonz/toonz/sources/toonzqt/styleselection.cpp:94:40: error: ordered comparison
      between pointer and zero ('TPalette::Page *' and 'int')
    if (palette->getStylePage(styleId) < 0) {
        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ ^ ~
1 error generated.
`
Did a little searching for getStylePage() to see what the expected output of this function is (I was guessing that this might be a QT function that changed with the newer Qt version?) and found this link which gave me a clue how to fix it:

https://www.viva64.com/en/b/0389/ (v503)

I was actually a little annoyed at the author of the article, in the time it took to write that article criticising the OpenToonz Code, he could have forked it, made the necessary changes and submitted a PR.

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,7 +1,6 @@
 language: cpp
 sudo: required
 dist: trusty
-group: deprecated-2017Q4 # temporary uses the previous version
 install:
     - if [[ $TRAVIS_OS_NAME == "osx" ]];   then bash ci-scripts/osx/travis-install.sh; fi
     - if [[ $TRAVIS_OS_NAME == "linux" ]]; then bash ci-scripts/linux/travis-install.sh; fi

--- a/toonz/sources/toonzqt/styleselection.cpp
+++ b/toonz/sources/toonzqt/styleselection.cpp
@@ -91,18 +91,13 @@ bool pasteStylesDataWithoutUndo(TPalette *palette, TPaletteHandle *pltHandle,
       break;
     }
 
-    if (palette->getStylePage(styleId) < 0) {
-      // styleId non e' utilizzato: uso quello
-      // (cut/paste utilizzato per spostare stili)
+    // For now styles will be inserted regardless the styleId of copied styles
+    // are already used in the target palette or not.
+    styleId = palette->getFirstUnpagedStyle();
+    if (styleId >= 0)
       palette->setStyle(styleId, style);
-    } else {
-      // styleId e' gia' utilizzato. ne devo prendere un altro
-      styleId = palette->getFirstUnpagedStyle();
-      if (styleId >= 0)
-        palette->setStyle(styleId, style);
-      else
-        styleId = palette->addStyle(style);
-    }
+    else
+      styleId = palette->addStyle(style);
 
     // check the type of the original(copied) style
     // If the original is NormalStyle

